pkg/insights/insightsclient/insightsclient.go

```diff
@@ -424,6 +428,11 @@ func ocmErrorMessage(url *url.URL, r *http.Response) error {
 	}
 }
 
+// IncrementRecvReportMetric increments the "insightsclient_request_recvreport_total" metric for the given HTTP status code
+func (c *Client) IncrementRecvReportMetric(statusCode int) {
```

Comment:
Does it need to be part of the `Client`? IMO, I'll make it just a function that receives the `metricsName` and the `statusCode`, and also makes it private to the package. What do you think?